ci-operator/step-registry/baremetalds/two-node/fencing/post-install/node-degredation/baremetalds-two-node-fencing-post-install-node-degredation-commands.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+set -o nounset -o errexit -o pipefail
+
+log(){ echo "[$(date +'%F %T%z')] $*"; }
```

Comment:
Lets' structure this a bit different to make it easier to follow. I would suggest we put the variables at the top and predefine them there. Then the helper functions ( also expand functions so they are not one liners, easier to read)